src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -43,19 +43,25 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
                 });
         }
 
-        internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        internal static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        {
+            return runtime.GetMetadataBlocks(appDomain, ImmutableArray<MetadataBlock>.Empty);
```

Comment:
Perhaps `default(ImmutableArray<MetadataBlock>)` for consistency with the caller of the new overload when there is no previous `CSharpMetadataContext`.